### ISSUE TITLE
No way to attach ARIA attributes to sliders

### ISSUE THREAD

**Issue by re-xyr** [huntabyte/shadcn-svelte#1810]
### Describe the bug

Currently, all props passed to the slider components get passed through into `bits-ui`'s `<Slider.Root>`; this is problematic for accessibility as it is the `<Slider.Thumb>` element that has the `slider` role (and therefore should receive ARIA attributes).

### Reproduction

Refer to: https://stackblitz.com/edit/github-xpdg2rta?file=src%2Froutes%2F%2Bpage.svelte

Observe that the `aria-label` is attached to the outermost `<div>` of the slider component, which does not have any ARIA role.

### System Info

```bash
System:
    OS: Linux 6.13 Arch Linux
    CPU: (20) x64 13th Gen Intel(R) Core(TM) i9-13900H
    Memory: 4.82 GB / 30.96 GB
    Container: Yes
    Shell: 5.2.37 - /usr/bin/bash
  Binaries:
    Node: 20.15.1 - /run/user/1000/fnm_multishells/63800_1743704855408/bin/node
    npm: 10.7.0 - /run/user/1000/fnm_multishells/63800_1743704855408/bin/npm
    bun: 1.2.8 - /usr/bin/bun
  Browsers:
    Chromium: 135.0.7049.52
  npmPackages:
    bits-ui: ^1.3.16 => 1.3.16 
    mode-watcher: ^0.5.1 => 0.5.1 
    svelte: ^5.25.6 => 5.25.6 
    svelte-sonner: ^0.3.28 => 0.3.28 
    tailwindcss: ^3.4.17 => 3.4.17
```

### Severity

annoyance

**Comment by re-xyr** [comment]
> You could always just add the label yourself that's the beautiful thing about owning the code.

Indeed that is what I did. But I would argue that the default template should promote correct behavior.

Or maybe since this project closely tracks shadcn-ui, I should first file an issue there? (They do also have this problem.)
